native/src/components/Header.tsx

```diff
@@ -210,8 +210,12 @@ const Header = ({
 
     const previousParams = previousRoute.params
 
-    // Poi details are not opened in a new route
-    if (route.name === POIS_ROUTE) {
+    if (
+      // Poi details are not opened in a new route
+      route.name === POIS_ROUTE &&
+      // unless coming from a deep link, and then the previous route has no slug
+      !!(previousParams as { slug?: string }).slug
```

Comment:
Hmm I am not entirely sure if we can make that assumption. A poi detail can also be linked to from a category or event and then the previous route has a slug :/

Reply:
Oh no, you're right! Back to the drawing board, I guess 😭

Reply:
Oh. Right, I overreacted. The slug trick does still work inside the Pois, just not in the header. Whew